test/unit/org/apache/cassandra/service/accord/AccordStateCacheTest.java

```diff
@@ -162,13 +185,15 @@ public void testAcquisitionAndRelease()
 
         Assert.assertSame(safeString1.global, cache.head());
         Assert.assertSame(safeString2.global, cache.tail());
+
+        assertCacheMetrics(cache.metrics, 0, 2, 2);
```

Comment:
The goal is just to test the global metrics here, correct?

Reply:
yes, I've just added also assertions against instance metrics